package.json

```diff
@@ -108,43 +108,16 @@
     "mock-fs": "5.2.0",
     "proxyquire": "2.1.3",
     "release-it": "17.10.0",
-    "remark-cli": "11.0.0",
-    "remark-frontmatter": "4.0.1",
+    "remark-cli": "12.0.1",
```

Comment:
Looks like we were already using remark, any context on that vs `prettier` which we use elsewhere?

Reply:
I was using `remark-lint` to lint my markdown files, to catch broken links, etc. I've not tried `prettier` with markdown, so not sure how it compares.